### PR TITLE
[Issue-7]: implement chat

### PR DESCRIPTION
**Pull request by swotex**
implement new font chat and backend manage

### CODE REVIEW

File: srcs/django_code/front/index.html [hunk 1]
```diff
@@ -10,14 +10,15 @@
 	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
 	
 	<!-- For font awesome -->
-	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> -->
```
Comment (Rreyth) [inline srcs/django_code/front/index.html:13]: comment

File: srcs/django_code/front/index.html [hunk 2]
```diff
@@ -10,14 +10,15 @@
...
 	<link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
 
 	<!-- Personnal css -->
 	<link rel="stylesheet" href="/static/css/common.css">
 	<link rel="stylesheet" href="/static/css/login.css">
-	<link rel="stylesheet" href="/static/css/minichat.css">
+	<!-- <link rel="stylesheet" href="/static/css/minichat.css"> -->
```
Comment (Rreyth) [inline srcs/django_code/front/index.html:19]: comment

**Review by Rreyth — CHANGES_REQUESTED** [review]
rm comments

File: srcs/django_code/front/index.html (hunk 2 above)
Comment (ltuffery) [reply srcs/django_code/front/index.html]: [64344f1](https://github.com/Rreyth/get-transcended/pull/58/commits/64344f1b7db42043771c5fe27b38e1dce395344c) fix ici

File: srcs/django_code/front/index.html (hunk 1 above)
Comment (ltuffery) [reply srcs/django_code/front/index.html]: [64344f1](https://github.com/Rreyth/get-transcended/pull/58/commits/64344f1b7db42043771c5fe27b38e1dce395344c) fix ici

File: srcs/django_code/front/components/minichat/setting.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+export class Setting extends Component {
+
+    static getName() {
+        return "setting"
+    }
+
+    connectedCallback() {
+        super.connectedCallback()
+
+        // const who = this.getAttribute('who') == "swotex" ? "msg-recieve" : "msg-sender"
```
[the remark this answers is not in the document]
Comment (ltuffery) [reply srcs/django_code/front/components/minichat/setting.js]: fix ici [64344f1](https://github.com/Rreyth/get-transcended/pull/58/commits/64344f1b7db42043771c5fe27b38e1dce395344c)

**Review by Rreyth — APPROVED** [review]
knuckles

**Pull request merged**